static/style.css

```diff
@@ -555,6 +548,58 @@
         border-left: 4px solid gray;
     }
 
+    button {
```

Comment:
This needs to be qualified -- it has made buttons within code blocks to copy the text nearly invisible